charts/neon-broker/values.yaml

```diff
@@ -0,0 +1,124 @@
+  # neon_region_slug: virginia
+
+# -- neon-broker's pods Security Context
+podSecurityContext: {}
+  # fsGroup: 2000
+
+# -- neon-broker's containers Security Context
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  # service.annotations -- Annotations to add to the service
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-type: external  # use newer AWS Load Balancer Controller
+    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
+    service.beta.kubernetes.io/aws-load-balancer-scheme: internal  # deploy LB to private subnet
+    # assign service to this name at external-dns
+    external-dns.alpha.kubernetes.io/hostname: neon-broker.local
+  # service.type -- Service type
+  type: LoadBalancer
+  # service.port -- broker listen port
+  port: 50051
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  #
+  # resources.limits -- The resources limits for the container
+  # resources.requests The requested resources for the container
+  #
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+# -- Node labels for pod assignment.
+nodeSelector: {}
+
+# -- Tolerations for pod assignment.
+tolerations: []
+
+# -- Affinity for pod assignment
+affinity: {}
+
+metrics:
+  # metrics.enabled -- Enable prometheus metrcis autodiscovery
+  enabled: false
+  ## Prometheus Operator ServiceMonitor configuration
+  ##
+  serviceMonitor:
+    # metrics.serviceMonitor.enabled -- Create ServiceMonitor resource
+    enabled: false
+
+    # metrics.serviceMonitor.namespace -- The namespace in which the ServiceMonitor will be created, if empty then Release.Namespace used
+    namespace: ""
+
+    ## Interval at which metrics should be scraped.
+    ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+    ##
+    # metrics.serviceMonitor.interval -- Interval in which prometheus scrapes
+    interval: 10s
+
+    ## Timeout after which the scrape is ended
+    ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+    ##
+    # metrics.serviceMonitor.scrapeTimeout -- Scrape Timeout duration for prometheus
+    scrapeTimeout: 10s
+
+    ## ServiceMonitor selector labels
+    ## ref: https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#prometheusioscrape
+    ##
+    # metrics.serviceMonitor.selector -- Additional labels to attach (used by Prometheus operator)
+    selector: {}
+    # selector:
+    #   prometheus: neon
```

Comment:
These are also very domain specific, so perhaps just leave out.

Reply:
Ok. BTW, isn't this is wrong that the value is called 'selector'? In reality these are labels attached to the ServiceMonitor.

Reply:
These indeed do go to `ServiceMonitor` labels. Unless I remember completely wrong, the prometheus operator we use from [kube-promeheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)  will select only `ServiceMonitor`s that have specific labels. So in this case it knows it's called `neon` , it would then select from labels `prometheus == neon`, if that makes sense? There could be more than one prometheus.

Reply:
Makes sense, though still not sure if 'selector' is optimal name.